src/widgets/section/SectionList.tsx

```diff
@@ -41,12 +48,105 @@ export default class SectionList extends WidgetBase<SectionListParameters> {
 		);
 	}
 
+	private _getContent() {
+		const { section, listIndex } = this.properties;
+		if (listIndex) {
+			return this.meta(Block).run(compiler)(section, listIndex, {
+				locale: i18n.locale
+			})
+		}
+
+		return null;
+	}
+
+	private _parseList(subsection: string, list?: DNode | DNode[]): DNode[] {
+		const { section } = this.properties;
+		if (!list) {
+			return [];
+		}
+
+		if (!Array.isArray(list)) {
+			list = [list];
+		}
+
+		let finalNodes: DNode[] = [];
+		for (let node of list) {
+			if (!node) {
+				continue;
+			}
+
+			if (isVNode(node)) {
+				if (node.tag === 'ul') {
+					node = <ul classes={css.list}>{node.children}</ul>;
+				}
+				else if (node.tag === 'p') {
+					finalNodes.push(...this._parseList(subsection, node.children));
+					continue;
+				}
+				else if (node.tag === 'a') {
+					const match = /.\/([\w\-\._~:\/?[\]@!\$&'\(\)\*\+,;=.]+)([#][\w\-\._~:\/?[\]@!\$&'\(\)\*\+,;=.]+)?$/g.exec(node.properties.href);
```

Comment:
phew this is a giant regex, what does it do?

Reply:
Yeah it was all of the possible characters allowed in a URL, but I don't think we need to be that specific. Will simplify.